fish-rust/src/builtins/return_exit.rs

```diff
@@ -0,0 +1,150 @@
+// Implementation of the return builtin.
+
```

Comment:
I don't think we should put both in the same file because that's inconsistent.
Rust makes it easy to share code across files.

I think we should name the files `return.rs` and `exit.rs`.
(we'll need to declare the module like `pub mod r#return;`)